frontend/javascripts/oxalis/view/right-border-tabs/tree_hierarchy_view.tsx

```diff
@@ -546,7 +582,6 @@ function TreeHierarchyView(props: Props) {
 
```

Comment:
yes, very good idea. the module was almost 1000 lines long and is now split into two halves. I put the render functions into a separate module as you suggested. I also hoisted the createMenu functions to the module scope. this made it more obvious that lots of handler-functions only need to be defined when the menu is created (instead of always when the node is visible). didn't measure that, but it's definitely better that way.

I'm afraid that commit is not really reviewable, though, since everything was moved around.. maybe have a final look at the two modules (instead of looking at the diff).